_posts/platform/deployment/2000-01-01-deploy-with-git.md

```diff
@@ -85,3 +85,56 @@ It will also add the `scalingo` remote to simplify future pushes.
   in any other way (GitLab / GitHub integration, archive, ...) the changes made
   to your code won't be available.
 {% endnote %}
+
+
+### Shallow Error
+
+Currently, Scalingo does not support update of shallowed repository (i.e. from a `git push`).
+The `shallow error` means that the pushed repository on Scalingo does not contains all its history.
+The error can happen whether you are pushing from your workstation or from an online worker.
+
+It is often the result of a clone with the Git parameter `--depth`).
```

Comment:
```suggestion
It is often the result of a clone with the Git parameter `--depth=1`.
```

Reply:
Any number is enough to make the repository a shallowed repo, that is why I didn't precise it.
Do you think it can lead to confusion to leave the 1 ?